HelpSource/Guides/UsingQuarks.schelp

```diff
@@ -5,7 +5,28 @@ related:: Classes/Quarks, Classes/Quark, Classes/Git
 
 Quarks are packages of SuperCollider code containing classes, extension methods, documentation and server UGen plugins. The Quarks class manages downloading these packages and installing or uninstalling them.
 
-section:: Installing quarks
+warning::
+Because individual users or groups develop their own Quark for different purposes, installing multiple packages can cause problems because they conflict with each other. We recommend that you select and install only the packages that you need. It is also recommended that you install one Quark package at a time to make it easier to troubleshoot problems as they arise.
+::
+
+note::
+If you have trouble finding the Quarks package you need using the Quarks installation methods in this document, you may find the Quarks package information in the following encyclopaedia useful:
+link::https://baryon.supercollider.online::
+::
+
+section::Preliminary stages
+
+To use Quark, you shoud install git software from link::https://git-scm.com::.
```

Comment:
"To use Quarks"

shoud --> should

Also, I have to admit that, reading through the first time, my eyes skipped right over the URL. Then I got confused by "the above webpage" in the OS-specific advice. So maybe this should be:

Quarks installation depends on emphasis::git:: version-control software. Detailed installation instructions may be found at link::https://git-scm.com::.

Reply:
thanks, done!